app/views/contexts/preview/preview_question.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Union
+
+from werkzeug.datastructures import ImmutableDict
+
+
+class PreviewQuestion:
+    def __init__(
+        self,
+        *,
+        block: ImmutableDict,
+    ):
+        self._block = block
+        self._block_id = block["id"]
+        self._question = self.get_question()
+        self._title = self._question["title"]
+        self._answers = self._build_answers()
+        self._descriptions = self._question.get("description")
+        self._guidance = self._question.get("guidance")
+        self._instruction = self._question.get("instruction")
+
+    def _build_answers(self) -> list[dict]:
+        answers_list = []
+        if answers := self._question.get("answers"):
+            for answer in answers:
```

Comment:
The indentation can be reduced by using `for answer in self._question.get("answers", []):`. Makes it easier to read.